cli/command/run/cmd.go

```diff
@@ -138,7 +176,15 @@ func mergeArgs(cfg *config.Project) {
 		cfg.Sauce.Region = defaultRegion
 	}
 
-	if region != "" {
-		cfg.Sauce.Region = region
+	if regionFlag != "" {
+		cfg.Sauce.Region = regionFlag
+	}
+
+	if cmd.Flags().Lookup("parallel").Changed {
+		cfg.Parallel = parallel
 	}
 }
+
+func enableCIProviders() {
+	github.Enable()
```

Comment:
same, having this in here seems to be too CI specific

Reply:
Actually, this was done on purpose.
The CI package is CI agnostic. Github, Gitlab & co. are specific CI implementations of our CI provider interface.
The run command package is but an entrypoint of the application. Exactly the perfect place where all dependencies should be constructed and passed further down.

FYI: I think long term, the CI runner will be moved out of the CI package and we'll stop referring to this as a "CI mode". It's more of a "I'm running within the image itself"-mode, for which we need a more apt name. This change will result in slightly different dependencies, at which point the interface will move as well and the enablement of CI providers can move into the CI package :D